src/server/scripts/EasternKingdoms/Karazhan/boss_nightbane.cpp

```diff
@@ -145,6 +146,10 @@ struct boss_nightbane : public BossAI
         {
             DoCastRandomTarget(SPELL_SEARING_CINDERS);
             context.Repeat(10s);
+        }).Schedule(1500ms, GROUP_GROUND, [this](TaskContext context)
+        {
+            DoCastVictim(SPELL_CLEAVE);
+            context.Repeat(1500ms, 45s);
```

Comment:
This timer feels off, how did you get to those values? 1,5sec delay means it will sometimes cast it twice in succession

Reply:
Sniffs from heyitsbench

Reply:
> 1,5sec delay means it will sometimes cast it twice in succession 

Can vouch that in my sniff there were two instances of Nightbane castimg cleave once and again ~1.5s later.